packages/web-react/src/components/UNSTABLE_Slider/__tests__/UNSTABLE_Slider.test.tsx

```diff
@@ -0,0 +1,48 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import UNSTABLE_Slider from '../UNSTABLE_Slider';
+
+describe('UNSTABLE_Slider', () => {
+  const id = 'slider-test';
+  const label = 'Slider';
+  const onChange = () => {};
+  const defaultValue = 10;
```

Comment:
```suggestion
  const DEFAULT_VALUE = 10;
```
?

Reply:
I looked in other test files and we have constants in camelCase, so I'll leave it like this.